packages/arb-ts/src/lib/L1ToL2Message/RetryableActions.ts

```diff
@@ -1,63 +0,0 @@
-import {
```

Comment:
Since we can instantiate from the l1 receipt of the L2 ticket creation receipt, I think I'm okay with this just being removed